hare/haretypes.go

```diff
@@ -53,6 +52,21 @@ func (mType MessageType) String() string {
 	}
 }
 
+func (id InstanceId) Id() uint32 {
```

Comment:
I dont understand why you need this, just use insanceId by itself without explicitly receiving it as uint32

Reply:
We use the interface Identifiable in order to be able to use different objects with maps. That way we don't need to work with concrete types.

Reply:
Fixed